webapp/channels/src/components/about_build_modal/about_build_modal.tsx

```diff
@@ -170,24 +170,17 @@ export default class AboutBuildModal extends React.PureComponent<Props, State> {
             </ExternalLink>
         );
 
-        // Only show build number if it's a number (so only builds from Jenkins)
```

Comment:
> only builds from Jenkins

🤣

Reply:
That old piece of code that follows you everywhere 😄